pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[build-system]
+requires = ["setuptools>=45", "setuptools-scm[toml]>=6.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "ts-interlockmonitor"
+description = "Code for handling TunableLaser's interlock-monitor system."
+license = { text = "GPL" }
+dependencies = ["aioserial"]
+readme = "README.rst"
+urls = { documentation = "https://ts-interlockmonitor.lsst.io", source_code = "https://github.com/lsst-ts/ts_interlockmonitor"}
+dynamic = ["version"]
```

Comment:
Shouldn't there be a project.scripts section for your entry point(s)?

Reply:
Please process this comment from Russell.

Reply:
Since there are no scripts now, it is not necessary to include the table.